rest_api/config.py

```diff
@@ -8,3 +8,5 @@
 
 LOG_LEVEL = os.getenv("LOG_LEVEL", "INFO")
 ROOT_PATH = os.getenv("ROOT_PATH", "/")
+
+CONCURRENT_REQUEST_PER_WORKER = os.getenv("CONCURRENT_REQUEST_PER_WORKER", 10)
```

Comment:
can we make default value to 4 otherwise 10 would be overkill for few CPUs

Reply:
✅